Resources/public/js/alloyeditor/buttons/embed.jsx

```diff
@@ -51,6 +51,20 @@ YUI.add('ez-alloyeditor-button-embed', function (Y) {
         },
 
         /**
+         * Checks if the command is disabled in the current selection.
+         *
+         * @method isDisabled
+         * @return {Boolean} True if the command is disabled, false otherwise.
+         */
+        isDisabled: function () {
```

Comment:
could do that, but then it won't work in `getStateClasses()` as expected by `ButtonStateClasses`

ref: https://github.com/liferay/alloy-editor/blob/25151898f9385f09096f05ab7cf30aeea281330d/src/ui/react/src/components/base/button-state-classes.js